wgsl/index.bs

```diff
@@ -12192,14 +12192,14 @@ For example, if `e` is a very small negative number, then `fract(e)` may be 1.0.
     <td>`T` is f32
   <tr>
     <td>Description
-    <td>Splits `e` into a significand and exponent so that `e` &equals; `significand * 2`<sup>`exponent`</sup>.
-    The significand is 0.0 or its magnitude is in the range [0.5, 1.0).
+    <td>Splits `e` into a fraction and exponent so that `e` &equals; `fraction * 2`<sup>`exponent`</sup>.
```

Comment:
(would it read better to convert these to include an article before exponent? "into a fraction and an exponent")

Reply:
Good point.  I've fixed that in the next commit.

Reply:
@dneto0 thank you!